### PR TITLE
fix: Fixes bullet chart issues

### PR DESCRIPTION
**Pull request by Golodhros**
## Description
Fixes issues with code and tests of the bullet chart

## Motivation and Context
Fixing stuff

## How Has This Been Tested?
Manually and through tests

## Types of changes
-   [ ] Refactor (changes the way we code something without changing its functionality)
-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/charts/bullet.js [hunk 1]
```diff
@@ -309,7 +306,7 @@ export default function module() {
                 .enter()
                 .append('line')
                 .attr('class', 'marker-line')
-                .attr('stroke', measureColor)
+                .attr('stroke', markerColor)
```
Comment (DalerAsrorov) [inline src/charts/bullet.js:309]: Are we sure we need to use marketColot instead? Just want to make sure that nothing looks off when we switch colors in demo

File: src/charts/bullet.js [hunk 2]
```diff
@@ -325,6 +322,10 @@ export default function module() {
      * @private
      */
     function drawHorizontalExtendedLine() {
+        if (baseLine) {
+            baseLine.remove();
```
Comment (DalerAsrorov) [inline src/charts/bullet.js:326]: Thanks I forgot to do all that last time

**Review by DalerAsrorov — APPROVED** [review]
Lgtm

File: src/charts/bullet.js (hunk 1 above)
Comment (Golodhros) [reply src/charts/bullet.js:309]: Yep, that way we can configure that color too. I think it was intended, but we missed it somehow.

**Pull request merged**